desktop-kde/kmail/autobuild/defines

```diff
@@ -3,13 +3,14 @@ PKGSEC=kde
 PKGDEP="akonadi akonadi-calendar akonadi-contacts akonadi-mime akonadi-search \
         kaccounts-integration kauth kcalcore kcalutils kcmutils kcodecs \
         kcompletion kconfig kconfigwidgets kcontacts kcoreaddons kcrash \
-        kdbusaddons ki18n kiconthemes kidentitymanagement kimap kio \
-        kitemmodels kitemviews kjobwidgets kldap kmailtransport kmime \
-        knotifications knotifyconfig kontactinterface kparts kpimtextedit \
-        kservice ktextwidgets ktnef kwallet kwidgetsaddons kxmlgui \
-        libgravatar libkdepim libksieve mailcommon messagelib pimcommon \
-        qtkeychain solid sonnet syntax-highlighting"
+        kdbusaddons kdepim-runtime ki18n kiconthemes kidentitymanagement \
+        kimap kio kitemmodels kitemviews kjobwidgets kldap kmailtransport \
+        kmime knotifications knotifyconfig kontactinterface kparts \
+        kpimtextedit kservice ktextwidgets ktnef kwallet kwidgetsaddons \
+        kxmlgui libgravatar libkdepim libksieve mailcommon messagelib \
+        pimcommon qtkeychain solid sonnet syntax-highlighting"
 BUILDDEP="extra-cmake-modules kdoctools"
+PKGRECOM="kmail-account-wizard"
```

Comment:
Missing `kmail-account-wizard` would cause errors when the user tries to add a new account in KMail. Add it to `PKGDEP` is probably a better idea.